base_export_security/models/export.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 LasLabs Inc.
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html).
+
+import datetime
+
+from odoo import _, api, fields, models
+from odoo.exceptions import AccessError
+
+
+class Export(models.Model):
+    _name = "export"
+    _description = 'Data Export Record'
+
+    name = fields.Char()
+    model_id = fields.Many2one(
+        'ir.model',
+        string='Exported Model',
+        readonly=True,
+    )
+    user_id = fields.Many2one(
+        'res.users',
+        string='Exported by',
+        readonly=True,
+    )
+    field_ids = fields.Many2many(
+        'ir.model.fields',
+        string='Exported Fields',
+        readonly=True,
+    )
+    record_ids = fields.Many2many(
+        'ir.model.data',
+        string='Exported Records',
+        readonly=True,
+    )
+
+    @api.model
+    def log_export(self, recordset, field_names):
+        date = datetime.datetime.now().strftime('%Y-%m-%d %H:%M:%S (UTC)')
+        model_name = recordset._name
+        model = self.env['ir.model'].search([('model', '=', model_name)])
+        user = self.env.user
+        name = '{} / {} / {}'.format(date, model.name, user.name)
```

Comment:
[Prefer % over .format(), prefer %(varname) instead of positional. This is better for translation and clarity.
](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#idioms)